From: Chris Renshaw <osm0sis@outlook.com>
Date: Fri, 16 Feb 2018 16:58:22 -0400
Subject: [PATCH] Add custom busybox version tagging

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ea9dd3d..c378593 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 1
 PATCHLEVEL = 28
 SUBLEVEL = 1
-EXTRAVERSION =
+EXTRAVERSION = -osm0sis
 NAME = Unnamed
 
 # *DOCUMENTATION*
-- 
2.5.3

