From: Chris Renshaw <osm0sis@outlook.com>
Date: Sat, 21 Jul 2018 18:36:56 -0300
Subject: [PATCH] Add custom busybox version tagging

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ea9dd3d..c378593 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 1
 PATCHLEVEL = 29
 SUBLEVEL = 1
-EXTRAVERSION =
+EXTRAVERSION = -osm0sis
 NAME = Unnamed
 
 # *DOCUMENTATION*
-- 
2.5.3

