From: Chris Renshaw <osm0sis@outlook.com>
Date: Fri, 4 Dec 2015 12:55:22 -0400
Subject: [PATCH] Fix util host "warning array subscript has type 'char'"

---
 scripts/kconfig/lxdialog/util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/kconfig/lxdialog/util.c b/scripts/kconfig/lxdialog/util.c
index 072d3ee..ab8b48c 100644
--- a/scripts/kconfig/lxdialog/util.c
+++ b/scripts/kconfig/lxdialog/util.c
@@ -347,7 +347,7 @@ int first_alpha(const char *string, const char *exempt)
 	int i, in_paren = 0, c;
 
 	for (i = 0; i < strlen(string); i++) {
-		c = tolower(string[i]);
+		c = tolower((unsigned char)string[i]);
 
 		if (strchr("<[(", c))
 			++in_paren;
-- 
2.5.3

