From: Chris Renshaw <osm0sis@outlook.com>
Date: Fri, 4 Dec 2015 12:49:56 -0400
Subject: [PATCH] Fix menubox host "warning array subscript has type 'char'"

---
 scripts/kconfig/lxdialog/menubox.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/kconfig/lxdialog/menubox.c b/scripts/kconfig/lxdialog/menubox.c
index 1fd501b..7292641 100644
--- a/scripts/kconfig/lxdialog/menubox.c
+++ b/scripts/kconfig/lxdialog/menubox.c
@@ -281,13 +281,13 @@ int dialog_menu(const char *title, const char *prompt, int height, int width,
 		else {
 			for (i = choice + 1; i < max_choice; i++) {
 				j = first_alpha(items[(scroll + i) * 2 + 1], "YyNnMmHh");
-				if (key == tolower(items[(scroll + i) * 2 + 1][j]))
+				if (key == tolower((unsigned char)items[(scroll + i) * 2 + 1][j]))
 					break;
 			}
 			if (i == max_choice)
 				for (i = 0; i < max_choice; i++) {
 					j = first_alpha(items [(scroll + i) * 2 + 1], "YyNnMmHh");
-					if (key == tolower(items[(scroll + i) * 2 + 1][j]))
+					if (key == tolower((unsigned char)items[(scroll + i) * 2 + 1][j]))
 						break;
 				}
 		}
-- 
2.5.3

