From: Chris Renshaw <osm0sis@outlook.com>
Date: Fri, 27 Nov 2015 10:04:50 -0400
Subject: [PATCH] Fix split-include host "warning array subscript has type 'char'"

---
 scripts/basic/split-include.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/basic/split-include.c b/scripts/basic/split-include.c
index e328788..64fffbe 100644
--- a/scripts/basic/split-include.c
+++ b/scripts/basic/split-include.c
@@ -116,7 +116,7 @@ int main(int argc, const char * argv [])
 	/* We found #define CONFIG_foo or #undef CONFIG_foo.
 	 * Make the output file name. */
 	str_config += sizeof(" CONFIG_") - 1;
-	for (itarget = 0; !isspace(str_config[itarget]); itarget++)
+	for (itarget = 0; !isspace((unsigned char) str_config[itarget]); itarget++)
 	{
 	    int c = (unsigned char) str_config[itarget];
 	    if (isupper(c)) c = tolower(c);
-- 
2.5.3

