From: Chris Renshaw <osm0sis@outlook.com>
Date: Tue, 30 Mar 2021 23:29:31 -0300
Subject: [PATCH] Fix possible "ld.bfd: cannot find -lrt" for older NDK toolchains

---
 Makefile.flags | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/Makefile.flags b/Makefile.flags
index cff935ff5..6d020e1ba 100644
--- a/Makefile.flags
+++ b/Makefile.flags
@@ -144,15 +144,24 @@ CFLAGS += --sysroot=$(CONFIG_SYSROOT)
 export SYSROOT=$(CONFIG_SYSROOT)
 endif
 
-# Android has no separate crypt library
+# Android has no separate crypt library, and only recently gained a separate rt library
 # gcc-4.2.1 fails if we try to feed C source on stdin:
 #  echo 'int main(void){return 0;}' | $(CC) $(CFLAGS) -lcrypt -o /dev/null -xc -
 # fall back to using a temp file:
 CRYPT_AVAILABLE := $(shell echo 'int main(void){return 0;}' >crypttest.c; $(CC) $(CFLAGS) -lcrypt -o /dev/null crypttest.c >/dev/null 2>&1 && echo "y"; rm crypttest.c)
+RT_AVAILABLE := $(shell echo 'int main(void){return 0;}' >rttest.c; $(CC) $(CFLAGS) -lrt -o /dev/null rttest.c >/dev/null 2>&1 && echo "y"; rm rttest.c)
 ifeq ($(CRYPT_AVAILABLE),y)
+ifeq ($(RT_AVAILABLE),y)
 LDLIBS += m rt crypt
 else
+LDLIBS += m crypt
+endif
+else
+ifeq ($(RT_AVAILABLE),y)
 LDLIBS += m rt
+else
+LDLIBS += m
+endif
 endif
 # libm may be needed for dc, awk, ntpd
 # librt may be needed for clock_gettime()
-- 
2.26.2

