From: Chris Renshaw <osm0sis@outlook.com>
Date: Fri, 16 Nov 2018 15:52:04 -0400
Subject: [PATCH] semctl: minor adjustments to support deprecated headers

---
 libbb/semctl.c                     |  2 +-
 1 file changed, 1 insertions(+), 1 deletions(-)

diff --git a/libbb/semctl.c b/libbb/semctl.c
index 5ddd07f3c..85183bb7b 100644
--- a/libbb/semctl.c
+++ b/libbb/semctl.c
@@ -22,7 +22,7 @@
 
 #include <sys/syscall.h>   /* For __NR_xxx definitions */
 #include <stdarg.h>
-#include <sys/sem.h>
+#include <linux/sem.h>
 #include "libbb.h"
 
 #ifndef __NR_semctl
