From 02a39a3f8cd1c0ef769d0d6f9b0ec24429917fe0 Mon Sep 17 00:00:00 2001
From: Tias Guns <tias@ulyssis.org>
Date: Tue, 27 Mar 2012 23:41:47 +0100
Subject: [PATCH] use BB_ADDITIONAL_PATH

---
 include/platform.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/platform.h b/include/platform.h
index 94d9f5a..a2c68fe 100644
--- a/include/platform.h
+++ b/include/platform.h
@@ -434,6 +434,7 @@ typedef unsigned smalluint;
 #endif
 
 #if defined(ANDROID) || defined(__ANDROID__)
+# define BB_ADDITIONAL_PATH ":/system/sbin:/system/bin:/system/xbin"
 # undef HAVE_DPRINTF
 # undef HAVE_GETLINE
 # undef HAVE_STPCPY
-- 
1.7.0.4

