From: Chris Renshaw <osm0sis@outlook.com>
Date: Tue, 11 Apr 2017 09:12:29 -0300
Subject: [PATCH] hack: fallback to default known dns server if __sys_property_get fails

---
 libres/dnscruft.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libres/dnscruft.c b/libres/dnscruft.c
index 142ffec..59e7fa2 100644
--- a/libres/dnscruft.c
+++ b/libres/dnscruft.c
@@ -73,7 +73,7 @@ void __dns_readstartfiles(void) {
   for(i = 1; i <= MAX_DNS_PROPERTIES; i++) {
     snprintf(propname, sizeof(propname), "%s%d", DNS_PROP_NAME_PREFIX, i);
     if(__system_property_get(propname, propvalue) < 1) {
-      break;
+      strcpy(propvalue,"8.8.8.8"); /* hack in a default nameserver since we seem to fail to get the property correctly */
     }
 
     if (parsesockaddr(propvalue,&_diet_res.nsaddr_list[_diet_res.nscount]))
-- 
2.8.3

